__tests__/components/map/UserCard.test.tsx

```diff
@@ -63,42 +66,42 @@ describe('<UserCard />', () => {
     screen.getByText(firstName + ' ' + lastName);
 
     // get phone number (throw error if no exist)
-    screen.getByText(phoneNumber);
+    screen.getByText(phone);
 
     // get initials of user (throw error if no exist)
     screen.getByText(`${firstName[0]}${lastName[0]}`);
   });
 
-  test('it should have functioning action buttons', () => {
-    // TODO: this is only to temporary mock alert() -- remove later
-    global.alert = jest.fn();
-
-    // create mock variables
-    let mockLocations: Location[] = [];
-
-    // TODO: mock the necessary functions for testing pressables
-    // (currently does not work)
-    const mockHandleStartNavigation = jest.fn(location =>
-      mockLocations.push(location)
-    );
-    const mockHandleCallUser = jest.fn();
-    const mockHandlePingUser = jest.fn();
-
-    // render component
-    render(<UserCard user={dummyUser} onClose={() => {}} />);
-
-    // get the pressables (throw error if no exist)
-    const press1 = screen.getByLabelText(
-      TestingLabel.USER_CARD_START_NAVIGATION
-    );
-    const press2 = screen.getByLabelText(TestingLabel.USER_CARD_CALL_USER);
-    const press3 = screen.getByLabelText(TestingLabel.USER_CARD_PING_USER);
-
-    // click on the pressables
-    fireEvent(press1, 'press');
-    fireEvent(press2, 'press');
-    fireEvent(press3, 'press');
-
-    // TODO: test what happens when each pressables are pressed
-  });
+  //   test('it should have functioning action buttons', () => {
+  //     // TODO: this is only to temporary mock alert() -- remove later
+  //     global.alert = jest.fn();
+
+  //     // create mock variables
+  //     let mockLocations: Location[] = [];
+
+  //     // TODO: mock the necessary functions for testing pressables
+  //     // (currently does not work)
+  //     const mockHandleStartNavigation = jest.fn(location =>
+  //       mockLocations.push(location)
+  //     );
+  //     const mockHandleCallUser = jest.fn();
+  //     const mockHandlePingUser = jest.fn();
+
+  //     // render component
+  //     render(<UserCard user={dummyUser} onClose={() => {}} />);
+
+  //     // get the pressables (throw error if no exist)
+  //     const press1 = screen.getByLabelText(
+  //       TestingLabel.USER_CARD_START_NAVIGATION
+  //     );
+  //     const press2 = screen.getByLabelText(TestingLabel.USER_CARD_CALL_USER);
+  //     const press3 = screen.getByLabelText(TestingLabel.USER_CARD_PING_USER);
+
+  //     // click on the pressables
+  //     fireEvent(press1, 'press');
+  //     fireEvent(press2, 'press');
+  //     fireEvent(press3, 'press');
+
+  //     // TODO: test what happens when each pressables are pressed
+  //   });
```

Comment:
why commented out?

Reply:
these are old tests. need to re-write in the future because our `<App/>` changed significantly since then